session/session.go

```diff
@@ -179,6 +190,22 @@ func (s *Session) Activate(ctx context.Context, i *identity.Identity, c lifespan
 	s.Identity = i
 	s.IdentityID = i.ID
 
+	agent := requestHeaders["User-Agent"]
+	if len(agent) > 0 {
+		s.UserAgent = strings.Join(agent, " ")
+	}
+
+	clientIP := requestHeaders.Get("True-Client-IP")
+	if clientIP != "" {
+		s.ClientIPAddress = clientIP
+	} else {
+		// TODO: Use x lib implementation to parse client IP address from the header string
+		s.ClientIPAddress = requestHeaders.Get("X-Forwarded-For")
+	}
```

Comment:
What about the "ClientIP" field in the request?

Reply:
Corrected the sequence by which the IP is derived